src/core/bpf/meson.build

```diff
@@ -13,7 +13,22 @@ clang_flags = [
         '-c',
 ]
 
-clang_arch_flag = '-D__@0@__'.format(host_machine.cpu_family())
+# Generate defines that are appropriate to tell the compiler what architecture
+# we're compiling for. By default we just map meson's cpu_family to __<cpu_family>__.
+# This dictionary contains the exceptions where this doesn't work.
```

Comment:
Maybe add a link for the cpu_family table to the comment so that it's clear where these come from:
https://mesonbuild.com/Reference-tables.html#cpu-families

Reply:
I added the comment now.